modules/fedimint-mint/src/lib.rs

```diff
@@ -265,22 +266,21 @@ impl FederationModuleConfigGen for MintConfigGenerator {
 )]
 pub struct MintInput(pub TieredMulti<Note>);
 #[autoimpl(Deref, DerefMut using self.0)]
-#[derive(Debug, Clone, Eq, PartialEq, Hash, Deserialize, Serialize, Encodable, Decodable)]
+#[derive(
+    Debug, Clone, Eq, PartialEq, Hash, Deserialize, Serialize, Encodable, Decodable, Default,
+)]
 pub struct MintOutput(pub TieredMulti<BlindNonce>);
 #[autoimpl(Deref, DerefMut using self.0)]
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Deserialize, Serialize, Encodable, Decodable)]
-pub struct MintOutputOutcome(pub Option<SigResponse>);
-#[autoimpl(Deref, DerefMut using self.0)]
-#[derive(Debug, Clone, Eq, PartialEq, Hash, Serialize, Deserialize, Encodable, Decodable)]
-pub struct MintConsensusItem(pub PartiallySignedRequest);
+pub struct MintOutputOutcome(pub Option<OutputOutcome>);
 
 #[async_trait(?Send)]
 impl ServerModulePlugin for Mint {
     type Decoder = MintModuleDecoder;
     type Input = MintInput;
     type Output = MintOutput;
     type OutputOutcome = MintOutputOutcome;
-    type ConsensusItem = MintConsensusItem;
+    type ConsensusItem = MintOutputConfirmation;
```

Comment:
That actually makes it less consistent

Reply:
Naming is the hardest.

Yeah, I was deliberating about it, and it's a second PR in this area that is making me think about it. I don't know if we already require 1:1 naming here, even if it leads to a newtype that just wraps another type that looks exactly the same, just with a different name. (`MintConsensusItem` vs `MintOutputConfirmation`)

It seems to me that as a priority we should strive for defining the domain (names that explain how things work). So as I read it, "the consensus items that the Mint module can create are always a confirmations to mint output from a peer (...)".

For some modules it will be necessary. Had there been multiple options of consensus items, it would make sense to put them in a:

```
enum MintConsensusItem {
   MintOutputConfirmation(...),
   MintSomethingSomethingElse,
}
```

 But do we always do it, just so the names match 1:1?

It might not be consistent, and possibly we should rename `MintOutput`/`MintInput` to more clear names. The problem was that `SignRequest` did not seem like a great domain name, but maybe it's just my PoV.

Reply:
`UnsignedNotes` (output) -> `NotesSignatureContribution` (consensus item) -> `NotesCreationOutcome` (outcome) -> `Notes` (input)?


I don't know. All I know is that Request/Response naming was weird. :D